pyproject.toml

```diff
@@ -32,7 +33,7 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.8,<4.0"
```

Comment:
Maybe just make it <3.12 to voice later version releases mess up without testing?